src/MonC/DotNetInterop/InteropResolver.cs

```diff
@@ -205,6 +209,11 @@ private void AddBinding(MethodInfo method, LinkableFunctionAttribute attribute,
                 }
             };
             _bindings[method.Name] = binding;
+
+            Symbol symbol = new Symbol();
+            symbol.Node = def;
+            symbol.SourceFile = method.Module.FullyQualifiedName;
```

Comment:
I can't figure out for the life of me how to access line information via the reflection API. Surely that data must be somewhere for debugging purposes.